content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,233 @@
+   $ cd examples/
+   ```
+
+### Create a New Stack
+
+Change the directory to `azure-cs-appservice` directory
+
+   ``` bash
+   $ cd azure-cs-appservice
+   ```
+
+There are four files in the Pulumi project, the `Azure.Appservice.csproj` which sets the .NET SDK and the Pulumi packages for the project. `Pulumi.yaml` holds project parameters such as name and runtime. The `Pulumi.dev.yaml` file holds the database password. `SharedAccessSignature.cs` contains a helper function to generate storage blob access URLs with SAS tokens in them. It demonstrates that you can add custom code inside your Pulumi program. `Program.cs` is the main program for declaring resources.
+
+1. Create a new stack:
+
+    ``` bash
+    $ pulumi stack init dev
+    ```
+
+1. Login to Azure CLI (you will be prompted to do this during deployment if you forget this step):
+
+    ``` bash
+    $ az login
+    ```
+
+1. Configure the location to deploy the resources to:
+
+    ``` bash
+    $ pulumi config set azure:location centralus
+    ```
+
+1. Define SQL Server password (make it complex enough to satisfy Azure policy):
+
+    ``` bash
+    pulumi config set --secret sqlPassword <value>
+    ```
+
+1. Run `pulumi up` to preview and deploy changes:
+
+    ``` bash
+    $ pulumi up
+    Previewing changes:
+    ...
+
+    Performing changes:
+    ...
+    info: 10 changes performed:
+
+        + 10 resources created
+
+    Update duration: 1m14.59910109s
+    ```
+
+___
+**NOTE:**
+
+> If you are using a free Azure account, some regions do not allow creating services if they are too busy. You might have to try a different [Azure location](https://azure.microsoft.com/en-us/global-infrastructure/locations/). You must tear down the stack resources to update the project's location information.
+___
+
+### Test Your Stack
+
+Check the deployed website endpoint by querying the stack and using curl to retrieve the web page.
+
+   ``` bash
+   $ pulumi stack output endpoint
+   https://azpulumi-as0ef47193.azurewebsites.net
+
+   $ curl "$(pulumi stack output endpoint)"
+   <html>
+      <body>
+         <h1>Greetings from Azure App Service!</h1>
+      </body>
+   </html>
+   ```
+
+From there, feel free to experiment. Simply making edits and running `pulumi up` will incrementally update your stack. Once finished, tear down your stack's resources by destroying and removing it:
+
+   ``` bash
+   $ pulumi destroy --yes
+   $ pulumi stack rm --yes
+   ```
+
+## Summary
+
+In this post, we've shown how to use Pulumi to deploy a modern application to Azure. The example code performs a number of tasks.
+
+First, we created a [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups), a [storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy-lrs), and an [application service plan](https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans) for our application:
+
+   ```C#
```

Comment:
We use `csharp` elsewhere, which works with the lang chooser. Best to just use `csharp` consistently throughout the website.

```suggestion
   ```csharp
```